README.md

```diff
@@ -1 +1,98 @@
-# etl
\ No newline at end of file
+# ETL
+
+The Supercluster Covid ETL is a NodeJS runnable program that:
+
+- **Extract**: collects the available DNAStack host data
+- **Transform**: converts this data into de-normalized sequence-centric documents
+- **Load**: Uploads this data to a new ElasticSearch index
+
+The application is written in TypeScript and gets built into a CLI.
+
+Each stage of the ETL can be run independently or in sequence through command line arguments, or through the npm scripts `extract`, `transform`, and `load` which are made available through the npm `package.json`.
+
+The application is also built into a Docker image available on ghcr.io (link to be added here once available).
+
+## Developer Quick Start
+
+Follow these steps to run the ETL in your local development environment, using docker for dependencies to run the ETL.
+
+1. Create `.env` file:
+
+   Copy the contents of `.env.example` to a new file named `.env` at the route of thise project. You can change any environment variables there as needed for your setup. The defaults provided will connect you to the DNAStack staging data api and the local dockerized services.
+
+1. Install NPM dependencies:
```

Comment:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/2107110/142060762-542b2092-d8fc-4b89-bd90-0097b1bd9a50.png">
Is this normal md syntax I'm not aware of? or just a github issue?

Reply:
Normal markdown. Is useful. ;)